### DISCUSSION TITLE
How to add a new line to a quote

### DISCUSSION THREAD

**Discussion by Brachyurus** [monkeytypegame/monkeytype#1244] [Questions] [answered]
I am trying to add a lithuanian quote that looks like this:
Jau saulelė vėl atkopdama budino svietą
Ir žiemos šaltos trūsus pargraudama juokės.
Šalčių pramonės su ledais sugaišti pagavo,
Ir putodams sniegs visur į nieką pavirto.

How do I make line breaks in the code?

**Comment by unrealapex** [comment, marked as answer]
Try using `\n` where you want to break a line. See this [Stack overflow answer](https://stackoverflow.com/questions/2392766/are-multi-line-strings-allowed-in-json) for more clarification

**Reply by Brachyurus** [reply]
Like this?
`"text": "Jau saulelė vėl atkopdama budino svietą\nIr žiemos šaltos trūsus pargraudama juokės.\nŠalčių pramonės su ledais sugaišti pagavo,\nIr putodams sniegs visur į nieką pavirto.",`

**Reply by unrealapex** [reply]
@Brachyurus  
Don't forget to mark an answer to a discussion if your question has been answered ;)
